### PR TITLE
SAR-10550 Mandatory fields marked as optional WCAG

### PR DESCRIPTION
**Pull request by jawilliams23**
***Awaiting content for Welsh messages - will create new sub-task***

### CODE REVIEW

File: app/views/CaptureInternationalAddress.scala.html [hunk 1]
```diff
@@ -32,7 +33,8 @@
         countries: Seq[String],
         submitAction: Call,
         headingKey: String,
-        name: Option[String] = None
+        name: Option[String] = None,
+        isPpob: Option[Boolean] = None
```
Comment (VLukovski) [inline app/views/CaptureInternationalAddress.scala.html:37]: doesn't need to be optional, in fact it shouldn't even work but thanks to twirl being weird it does

File: app/views/CaptureInternationalAddress.scala.html [hunk 2]
```diff
@@ -111,7 +103,40 @@
...
 
+}
+
+@postcodeBlock = @{
+    if (isPpob) ppobBlock
+    else nonPpobBlock
+}
+
+@ppobBlock = {
+    @inputTextHintNonBold(
```
Comment (VLukovski) [inline app/views/CaptureInternationalAddress.scala.html:119]: the way this component displays the heading and the hint is very similar, did we check with design if this is how it's meant to be or do we need to change something else.

File: conf/messages [hunk 3]
```diff
@@ -145,7 +145,8 @@ international.addressLookup.homeAddress.editPage.line1Label                = Add
 international.addressLookup.homeAddress.editPage.line2Label                = Address line 2
 international.addressLookup.homeAddress.editPage.line3Label                = Address line 3 (optional)
 international.addressLookup.homeAddress.editPage.townLabel                 = Town/City
-international.addressLookup.homeAddress.editPage.postcodeLabel             = Postcode (optional)
+international.addressLookup.homeAddress.editPage.postcodeLabel             = Postcode
```
Comment (VLukovski) [inline conf/messages:148]: Do we need to be updating ALF content? In fact we do not even send postcodeHints there.

File: app/views/CaptureInternationalAddress.scala.html (hunk 2 above)
Comment (jawilliams23) [reply app/views/CaptureInternationalAddress.scala.html:119]: May need to be a change - I'm not sure

**Pull request merged**